packages/grafana-ui/src/Intro.story.mdx

```diff
@@ -1,5 +1,32 @@
-<Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false}, id: 1}}/>
+<Meta title="Docs Overview/Intro" parameters={{ options: { isToolshown: false }, id: 1 }} />
 
-# @grafana/ui
+# @grafana/ui components library
 
-Welcome to @grafana/ui
+> **@grafana/ui is currently in BETA**. Core API is unstable and can be subject to changes!
+
+@grafana/ui is a collection of components used by [Grafana](https://github.com/grafana/grafana)
+
+Our goal is to deliver Grafana's common UI elements for plugins developers and contributors.
+
+See [package source](https://github.com/grafana/grafana/tree/master/packages/grafana-ui) for more details.
+
+## Installation
+
+`yarn add @grafana/ui`
+
+`npm install @grafana/ui`
+
+## Use Storybook to explore the components
+
+Storybook is used to host example implementations of the components in `@grafana/ui`.
+
+The `Canvas` tab can be used for:
+
+- Changing props for the component in the Knobs tab
+- Seeing code implementation in the Story tab
+
+The `Docs` tab provides:
+
+- Written docs with description on how to use the component
+- A props table
+- Code examples
```

Comment:
Great!


Can you add also section with links to grafana documentation and tutorials? Like here https://github.com/grafana/simple-react-panel

Reply:
Awesome! Also got some feedback from @marcusolsson that I'm going to implement